teuthology/task/ceph_ansible.py

```diff
@@ -210,6 +215,56 @@ def teardown(self):
         os.remove(self.inventory)
         os.remove(self.playbook_file)
         os.remove(self.extra_vars_file)
+        machine_type = self.ctx.config('machine_type')
```

Comment:
This is not going to be reliable. You'll probably want to use ``teuthology.lock.query.is_vm`` or ``Remote.is_vm``